### DISCUSSION TITLE
Lazer detects my tablet as touch device

### DISCUSSION THREAD

**Discussion by RostyslawRonevych** [ppy/osu#25970] [Q&A] [answered]
Im using Gaomon S620 tablet for osu
Today I installed Lazer and it thinks im using a touch device and applies TD mod, did amyone else experience the same?
![image](https://github.com/ppy/osu/assets/62483964/e7c6da3d-b5c3-4eb1-adb3-d69e4545b977)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/25590

**Reply by RostyslawRonevych** [reply]
Thanks!
